server/sonar-server/src/main/java/org/sonar/server/setting/ws/SetAction.java

```diff
@@ -288,7 +288,7 @@ private static SetRequest toWsRequest(Request request) {
     try {
       return (Map<String, String>) gson.fromJson(json, type);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "Map". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1905)